core/dbt/contracts/graph/parsed.py

```diff
@@ -229,6 +228,11 @@ class ParsedNode(ParsedNodeDefaults, ParsedNodeMixins, SerializableType):
     def _serialize(self):
         return self.to_dict()
 
+    def __post_serialize__(self, dct):
+        if 'config_call_dict' in dct:
+            del dct['config_call_dict']
+        return dct
```

Comment:
I don't really understand what's happening here.

Reply:
We don't want the 'config_call_dict' in the serialized output. We needed to save it for when we reprocess the config dictionaries, but it shouldn't be saved in artifacts or the partial parse manifest.

Reply:
ahh that makes sense. Thanks for the explanation. I think a comment to that effect would help.